csfieldguide/chapters/content/en/complexity-and-tractability/sections/tractability.md

```diff
@@ -52,7 +52,7 @@ It's still completely impractical!
 
 Another way to solve problems faster is to have multiple processors work on different solutions at the same time.
 If you were to buy 1,000 processors (e.g. 1,000 computers, or 250 4-core computers) and have each one test out different routes, then the solution could be found 1,000 times faster.
-Try changing the number of processors to 1,000, and see how long that would take (you may need to change the units back — is it seconds? hours? days?)
+Try changing the number of processors to 1,000 and see how long that would take (you may need to change the units back). Is it seconds? Hours? Days?
```

Comment:
`Is it seconds? Hours? Days?` can stay on the same line, but it should be on a new one